pkg/snowflake/generic.go

```diff
@@ -81,6 +87,10 @@ func (ab *AlterPropertiesBuilder) Statement() string {
 		sb.WriteString(fmt.Sprintf(" %s=%d", strings.ToUpper(k), v))
 	}
 
+	for k, v := range ab.floatProperties {
+		sb.WriteString(fmt.Sprintf(" %s=%f", strings.ToUpper(k), v))
```

Comment:
```suggestion
		sb.WriteString(fmt.Sprintf(" %s=%2f", strings.ToUpper(k), v))
```
what do you think about setting the resolution to 2 decimals?

Reply:
if that's what users usually provide I think it makes more sense to round up to 2 figures. I'll fix the tests and documentation to reflect that.